engines/wintermute/base/gfx/opengl/base_surface_opengl3d.cpp

```diff
@@ -234,6 +243,9 @@ bool BaseSurfaceOpenGL3D::putSurface(const Graphics::Surface &surface, bool hasA
 
 	if (_imageData && _imageData != &surface) {
 		_imageData->copyFrom(surface);
+
+		// TODO: Use OpenGL features to apply the mask?
```

Comment:
About the comment.I would like not ot change common 2D path in 3D renderer. I think it would be better specialize video playback path. Still at that stage of 3D enabling, I want avoid that kind of optimisaztions.

Reply:
OK, I've removed the comment.